js/about/preferences.js

```diff
@@ -137,17 +136,7 @@ class GeneralTab extends ImmutableComponent {
       )
     })
 
-    let homepageValue = getSetting(settings.HOMEPAGE, this.props.settings)
-    if (typeof homepageValue === 'string') {
-      const punycodeUrl = UrlUtil.getPunycodeUrl(homepageValue)
-      if (punycodeUrl.replace(/\/$/, '') !== homepageValue) {
-        homepageValue = UrlUtil.getPunycodeUrl(homepageValue)
-      }
-
-      // we use | as a separator for multiple home pages
-      homepageValue = homepageValue.replace(/%7C/g, '|')
-    }
-    const homepage = homepageValue && homepageValue.trim()
+    const homepage = getSetting(settings.HOMEPAGE, this.props.settings)
```

Comment:
I think removing the punycode encoding may cause a regression

Can you please test this with an international domain name?
http://www.i18nguy.com/markup/idna-examples.html

Reply:
Punycode was just moved from component into appState. Please look at line 142 in appState

Reply:
https://github.com/brave/browser-laptop/pull/10659/files#diff-23ca389e2bcb77191b5a9c10900eb3a3R142

Reply:
ah- totally missed that 😄  perfect!